ublox/src/ubx_packets/packets.rs

```diff
@@ -1020,6 +1159,163 @@ impl<T: FloatCore + FromPrimitive + ToPrimitive> ScaleBack<T> {
     }
 }
 
+/// Navigation Engine Expert Settings
+#[ubx_packet_recv_send]
+#[ubx(
+    class = 0x06,
+    id = 0x23,
+    fixed_payload_len = 40,
+    flags = "default_for_builder"
+)]
+struct CfgNavX5 {
+    /// Only version 2 supported
+    version: u16,
+
+    /// Only the masked parameters will be applied
+    #[ubx(map_type = CfgNavX5Params1)]
+    mask1: u16,
+
+    #[ubx(map_type = CfgNavX5Params2)]
+    mask2: u32,
+
+    /// Reserved
+    reserved1: [u8; 2],
+
+    /// Minimum number of satellites for navigation
+    min_svs: u8,
+
+    ///Maximum number of satellites for navigation
+    max_svs: u8,
+
+    /// Minimum satellite signal level for navigation
+    min_cno_dbhz: u8,
+
+    /// Reserved
+    reserved2: u8,
+
+    /// initial fix must be 3D
+    ini_fix_3d: u8,
+
+    /// Reserved
+    reserved3: [u8; 2],
+
+    /// issue acknowledgements for assistance message input
+    ack_aiding: u8,
+
+    /// GPS week rollover number
+    wkn_rollover: u16,
+
+    /// Permanently attenuated signal compensation
+    sig_atten_comp_mode: u8,
+
+    /// Reserved
+    reserved4: u8,
+    reserved5: [u8; 2],
+    reserved6: [u8; 2],
```

Comment:
Why do you separate this into three different fields? (rather than `[u8; 5]`)

Reply:
So this is how it is defined in the specification. This is likely just an undocumented field. If someone wanted to RE these fields, at least they are pulled out correctly here.